dev-system/src/cmr/dev_system/system.clj

```diff
@@ -97,6 +97,9 @@
   [system]
   (update-in system [:apps] set-web-server-options))
 
+(def in-memory-elastic-log-level-atom
```

Comment:
Can we have a TODO or something to remind us to turn this off eventually

Reply:
I'm not sure i understand what you mean?

Reply:
We don't want to leave INFO on? pretty straight forward.